docs/changelog.mdx

```diff
@@ -312,7 +312,7 @@ us at info@regression.gg for dedicated support and access.
 
 - [Agent Builder](404) Beta - Our GPT-enabled agent builder product is live for select players, now with improved validation tools. If you&apos;d like to try it out, let us know!
 - Our documentation site is now live! This docs site includes API/SDK references, tutorials, guides, and FAQs for both players and developers.
-- [API Keys](getting-started/api-keys/authenticating-with-api-keys) - developers and players can now create API keys to interact with RG services
+- [API Keys](core-concepts/authenticating-with-api-keys) - developers and players can now create API keys to interact with RG services
```

Comment:
getting-started/installing-regression-games already has a section about the api-keys in it.

I don't want to have to maintain that content in 2 places.  Can we either inline this new content there or do something so that we don't have 2 separate writeups on similar content

Reply:
Created two partials for "create key" and "set key in unity". This is what the relevant section of the installation guide looks like now

![image](https://github.com/user-attachments/assets/b9d87363-bc4c-4ebb-86da-10bcf3709d49)